src/rtextures.c

```diff
@@ -4985,6 +4985,27 @@ Vector3 ColorToHSV(Color color)
     return hsv;
 }
 
+/*
```

Comment:
Please, could you review the comment to be consistent with the other color functions. No functions use that many comment lines and neither /* */ comments.